libraries/helpers.rb

```diff
@@ -34,15 +34,29 @@ def ingredient_package_name
       product_lookup(new_resource.product_name, version_string(new_resource.version))['package-name']
     end
 
-    def install_mixlib_versioning
-      # We need Mixlib::Versioning in the library helpers for
-      # parsing the version string.
-      chef_gem "#{new_resource.product_name}-mixlib-versioning" do # ~FC009 foodcritic needs an update
-        package_name 'mixlib-versioning'
-        compile_time true
+    def ensure_mixlib_versioning_gem_installed!
+      node.run_state[:mixlib_versioning_gem_installed] ||= begin # ~FC001
+        install_gem_from_rubygems('mixlib-versioning', '1.1.0')
+
+        require 'mixlib/versioning'
+        true
       end
+    end
 
-      require 'mixlib/versioning'
+    def ensure_mixlib_install_gem_installed!
+      node.run_state[:mixlib_install_gem_installed] ||= begin # ~FC001
+        install_gem_from_rubygems('mixlib-install', '0.8.0.alpha.0')
```

Comment:
I presume the version is specified because we're using a prerelease gem? Do we want to support options in the `install_gem_from_rubygems` and pass in `--pre`? Not sure what the best way to handle this is - I don't know if we want to hardcode an alpha version here. When would we update it? Why can't we release 0.8.0 and use the latest?